assignment1/Buynow.java

```diff
@@ -0,0 +1,180 @@
+package assignment1;
+
+import java.util.Scanner;
+
+public class Buynow {
```

Comment:
Can use a proper name of this class, this class is the entry point of the program, we should name it accordingly